src/main/java/com/minecolonies/entity/ai/minimal/EntityAIGoHome.java

```diff
@@ -1,10 +1,15 @@
 package com.minecolonies.entity.ai.minimal;
 
 import com.minecolonies.entity.EntityCitizen;
+import com.minecolonies.sounds.FishermanSounds;
 import com.minecolonies.util.Log;
+import com.minecolonies.util.SoundUtils;
 import net.minecraft.entity.ai.EntityAIBase;
+import net.minecraft.util.SoundEvent;
 import net.minecraft.util.math.BlockPos;
 
+import java.util.Random;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.Random'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)